lib/wibox/layout/grid.lua

```diff
@@ -0,0 +1,712 @@
+---------------------------------------------------------------------------
+--
+--
+--
+--@DOC_wibox_widget_defaults_grid_EXAMPLE@
+-- @author getzze
+-- @copyright 2017 getzze
+-- @classmod wibox.layout.grid
+---------------------------------------------------------------------------
+
+local setmetatable = setmetatable
+local unpack = unpack or table.unpack -- luacheck: globals unpack (compatibility with Lua 5.1)
+local table = table
+local pairs = pairs
+local ipairs = ipairs
+local math = math
+local util = require("awful.util")
+local base = require("wibox.widget.base")
+local beautiful = require("beautiful")
+
+
+local grid = { mt = {} }
+
+local properties = {
+                    "orientation", "forced_num_rows", "forced_num_cols",
+                    "min_cols_size", "min_rows_size",
+                    --"max_cols_size", "max_rows_size",
+                   }
+
+local dir_properties = { "spacing", "padding", "fixed_size", "expand" }
+
+
+--- Reset a ratio layout. This removes all widgets from the layout.
```

Comment:
Also, to me this sounds like it just removes all widgets. How about `Among other things, this removes all widgets from the layout`?